templates/certified/desktops-search-results.html

```diff
@@ -0,0 +1,311 @@
+{% extends "certified/base_certification.html" %}
+
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
+
+{% block content %}
+
+<form name="searchResults">
+ <section class="p-strip--suru-topped">
+   <div class="u-fixed-width">
+     <h1 class="u-sv3">Ubuntu certified
```

Comment:
Could you please remove the logic here and just have Ubuntu certified desktops for now? I'll be updating the top section of this page but for now it makes sense to have desktops in the title if you're at `/certified/desktops`

Reply:
Good point. Done